.github/workflows/build.yml

```diff
@@ -26,12 +26,14 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Setup .NET
-      uses: actions/setup-dotnet@v1
+      uses: actions/setup-dotnet@v2
       with:
+        include-prerelease: true
         dotnet-version: | 
           3.1.x
           5.0.x
           6.0.x
+          7.0.x
```

Comment:
on `-latest` I guess most of these are already available, might not need setup step at all: https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#net-core-sdk

Reply:
I saw that in the logs of the action, was not sure

Reply:
*takes out the black book and prepares to preach...* Have I told you about NUKE build? 😆

Reply:
What is NUKE, sounds interesting 🕺

Reply:
Is it my turn to say... Go to hell Sebastien 😆